From 8c2c51b66d69ce781efafc3a1e47e91c702890b9 Mon Sep 17 00:00:00 2001
From: Rodney Dawes <dobey.pwns@gmail.com>
Date: Fri, 10 Jun 2022 13:18:05 -0400
Subject: [PATCH 01/13] Remove unused and useless install_${PROJECT_NAME}
 target

Signed-off-by: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 07ef083a..5e81bcf7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -465,8 +465,3 @@ if(enable-tests)
             MALIIT_PLUGINS_DATADIR=${CMAKE_SOURCE_DIR}/data)
 
 endif()
-
-add_custom_target(install_${PROJECT_NAME}
-        ${CMAKE_BUILD_TOOL} install
-        DEPENDS ${PROJECT_NAME}
-        COMMENT "Installing ${PROJECT_NAME}")
-- 
2.30.2

