From 8490478dce357bdfcd3d84afa74d7756409e910d Mon Sep 17 00:00:00 2001
From: Rodney Dawes <dobey.pwns@gmail.com>
Date: Fri, 10 Jun 2022 14:21:03 -0400
Subject: [PATCH 02/13] cmake: Fix include paths and linking of tests

Signed-off-by: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
---
 CMakeLists.txt | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5e81bcf7..441e79ff 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -427,12 +427,11 @@ if(enable-tests)
     add_library(test-utils STATIC
             tests/unittests/common/inputmethodhostprobe.cpp
             tests/unittests/common/inputmethodhostprobe.h
-            tests/unittests/common/utils-gui.cpp
             tests/unittests/common/utils.cpp
             tests/unittests/common/utils.h)
 
-    target_link_libraries(test-utils PUBLIC Qt5::Core Qt5::Gui Qt5::Test)
-    target_include_directories(test-utils INTERFACE tests/common tests maliit-keyboard)
+    target_link_libraries(test-utils PUBLIC Maliit::Plugins Qt5::Core Qt5::Gui Qt5::Test westernsupport)
+    target_include_directories(test-utils PUBLIC tests/unittests tests/unittests/common src src/lib)
 
     function(create_test name)
         set(_extra_sources ${ARGV})
@@ -442,7 +441,7 @@ if(enable-tests)
                 tests/unittests/${name}/${name}.cpp
                 ${_extra_sources})
 
-        target_link_libraries(${name} test-utils ${_libs})
+        target_link_libraries(${name} test-utils maliit-keyboard-common ${_libs})
 
         add_test(${name} ${name})
 
-- 
2.30.2

