diff --git a/src/3rdparty/QGSettings/qconftypes.cpp b/src/3rdparty/QGSettings/qconftypes.cpp
index 805a41f5..8c9a4072 100644
--- a/src/3rdparty/QGSettings/qconftypes.cpp
+++ b/src/3rdparty/QGSettings/qconftypes.cpp
@@ -230,7 +230,7 @@ GVariant *qconf_types_collect_from_variant(const GVariantType *gtype, const QVar
             gsize size = array.size();
             gpointer data;
 
-            data = g_memdup2(array.data(), size);
+            data = g_memdup(array.data(), size);
 
             return g_variant_new_from_data(G_VARIANT_TYPE_BYTESTRING,
                                            data, size, TRUE, g_free, data);
